base/tps/shared/conf/CS.cfg

```diff
@@ -80,7 +80,7 @@ auths.instance.ldap1.externalReg.cuidAttributeName=tokenCUID
 auths.instance.ldap1.externalReg.registrationTypeAttributeName=registrationtype
 auths.instance.ldap1.externalReg.tokenTypeAttributeName=tokenType
 auths.instance.ldap1.ldap.maxConns=15
-auths.instance.ldap1.ldap.minConns=3
+auths.instance.ldap1.ldap.minConns=0
```

Comment:
The minConns could have an  impact on the execution time because with 0 the system never keep idle connection ready to use. In future we have to delay the connection pool creation without modify this parameter.

Reply:
This is actually just the default/initial value and the admin is expected to fine-tune the system for their environment, but I agree that the code should work well regardless of the value. I've updated the PR to set the param in `pki-tps-run` instead so it will only affect TPS containers. We can do a further investigation on this later.